backend/api/webhooksHandler.go

```diff
@@ -13,13 +13,22 @@ func publishMqtt(mqttServer *server.Server) gin.HandlerFunc {
 	return func(c *gin.Context) {
 		go func() {
 			data, _ := c.Value("data").(*gabs.Container)
+			event, err := gabs.ParseJSON(data.Bytes());
+
+			if err != nil {
+				fmt.Println("Error again", event, err)
+				return
+			}
+
+			event.Delete("secret_key")
```

Comment:
Let's avoid creating `event` here, since data can be manipulated directly.
Try to `data.Delete("secret_key")` instead to see what happens. :smile:

Reply:
fixed